src/main/java/org/spongepowered/api/data/manipulator/mutable/entity/CustomNameVisibleData.java

```diff
@@ -42,6 +43,8 @@
      * Returns whether the display name is visible to players.
      *
      * @return Whether the display name is visible or not
+     *
```

Comment:
Shouldn't be a newline between other javadoc tags and `@see`. Applies to all cases you've done this.